Sources/Danger/GitHubDSL.swift

```diff
@@ -344,7 +344,7 @@ public struct GitHubCommit: Decodable {
     public let url: String
 
     /// The GitHub user who wrote the code.
-    public let author: GitHubUser
+    public let author: GitHubUser?
```

Comment:
It seems like there were some API changes by GitHub. If you take a look at the latest Travis build
```
danger process .build/x86_64-apple-macosx10.10/debug/danger-swift
```
fails because in the data danger received from GitHub author and committer are null.

Reply:
When calling the api directly the values author and comitter are null: https://api.github.com/repos/danger/danger-swift/pulls/66/commits